include/snowflake/platform.h

```diff
@@ -104,6 +104,10 @@ void STDCALL sf_os_version(char *ret);
 
 int STDCALL sf_strncasecmp(const char *s1, const char *s2, size_t n);
 
+char *STDCALL sf_filename_from_path(const char *path);
```

Comment:
Excluding the directory name, because it will include the build environment directory name.
The filename should be good enough as long as the namespace is distinguishable.